content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md

```diff
@@ -59,8 +59,11 @@ Autoscaler. The monitoring pipeline fetches metrics from the kubelet and
 then exposes them to Kubernetes via an adapter by implementing either the
 `custom.metrics.k8s.io` or `external.metrics.k8s.io` API. 
 
-[Prometheus](https://prometheus.io), a CNCF project, can natively monitor Kubernetes, nodes, and Prometheus itself.
-Full metrics pipeline projects that are not part of the CNCF are outside the scope of Kubernetes documentation.  
+Kubernetes is designed to work with [OpenMetrics](https://openmetrics.io/) which is one of the CNCF Project. 
```

Comment:
👋 @NitishKumar06 . What do you think about these edits:

Kubernetes is designed to work with [OpenMetrics](https://openmetrics.io/) which is one of the [CNCF Observability and Analysis - Monitoring Projects](https://landscape.cncf.io/card-mode?category=monitoring&project=graduated,incubating,member,no&grouping=category&sort=stars). Full metrics pipeline projects that are not part of the CNCF are outside the scope of the Kubernetes documentation.